src/components/chatbox/chatbox.module.css

```diff
@@ -21,6 +21,13 @@
 }
 
 .arrowdown {
+	/* button overrides - start */
+	height: 28px;
+	width: 28px;
```

Comment:
You can pass `size="small"` to get a similar height (30px) and avoid this override.